README.md

```diff
@@ -5,23 +5,23 @@ This may include frameworks with a browser-side component.
 
 Only frameworks with over 1,000 stars are currently added.
 Feel free to add your own.
-The frameworks are listed by number of watchers, descending, as of March 18, 2014.
+The frameworks are listed by number of watchers, descending, as of February 9, 2016.
 
 | Framework              | Sponsor(s) / Author(s)                                | Watchers | Stars  |
-------------------------:|-------------------------------------------------------|----------|--------|
-| [Meteor][meteor]       | [Meteor Development Group][meteorgroup]               | 973      | 11,687 |
-| [Express][express]     | [StrongLoop][strongloop] / [TJ Holowaychuk][tj]       | 828      | 12,824 |
-| [Sails][sails]         | [Mike McNeil][mikemcneil]                             | 485      | 5,582  |
-| [Koa][koa]             | [TJ Holowaychuk][tj]                                  | 232      | 3,160  |
-| [Derby][derby]         | [Nate Smith][natesmith] [Brian Noguchi][briannoguchi] | 213      | 3,079  |
-| [Kraken][kraken]       | [PayPal][paypal]                                      | 210      | 2,424  |
-| [Hapi][hapi]           | [Walmart Labs][walmart]                               | 204      | 1,981  |
-| [Connect][connect]     | [Sencha Labs][senchalabs]                             | 202      | 4,180  |
-| [LoopBack][loopback]   | [StrongLoop][strongloop]                              | 187      | 2,602  |
-| [Restify][restify]     | [Mark Cavage][markcavage]                             | 122      | 2,001  |
-| [Geddy][geddy]         | [Matthew Eernisse][mattheweernisse]                   | 101      | 1,703  |
-| [CompoundJS][compound] | [Anatoliy Chakkaev][1602]                             | 91       | 1,542  |
-| [Flatiron][flatiron]   | [Nodejitsu][nodejitsu]                                | 75       | 1,146  |
+|-----------------------:|-------------------------------------------------------|----------|--------|
+| [Meteor][meteor]       | [Meteor Development Group][meteorgroup]               | 1,864    | 31,918 |
+| [Express][express]     | [StrongLoop][strongloop] / [TJ Holowaychuk][tj]       | 1,344    | 22,945 |
+| [Sails][sails]         | [Mike McNeil][mikemcneil]                             | 791      | 13,405 |
+| [Koa][koa]             | [TJ Holowaychuk][tj]                                  | 527      | 8,862  |
+| [Hapi][hapi]           | [Walmart Labs][walmart]                               | 358      | 5,449  |
```

Comment:
I only updated the stats. That info is just as wrong as it is in the current version. My commit only changes the ordering.